core/src/test/java/tachyon/UserInfoTest.java

```diff
@@ -26,17 +26,7 @@
   private static final int DELTA = 50;
 
   private static final String USERINFOR_TOSTRING =
-      "UserInfo( mUserId: 99, mOwnBytes: 1064, mLastHeartbeatMs: ";
-
-  @Test
-  public void addOwnBytesTest() {
```

Comment:
user own bytes will be maintained in StorageDir, there will be no ownBytes in Users anymore.